scrapy/settings/__init__.py

```diff
@@ -37,20 +37,34 @@ class SettingsAttribute:
     """
 
     def __init__(self, value, priority):
-        self.value = value
+        self._value = value
         if isinstance(self.value, BaseSettings):
-            self.priority = max(self.value.maxpriority(), priority)
+            self.priority = max(self._value.maxpriority(), priority)
         else:
             self.priority = priority
+        self.hit = False
```

Comment:
I wonder if there’s a better word for this, as to me `hit` could be both for setting and getting a setting value. `read` could be mistaken by a verb. `has_been_read`?